core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-background-columns/template-builder-background-columns.component.ts

```diff
@@ -10,16 +10,18 @@ import { GRID_STACK_MARGIN_HORIZONTAL, GRID_STACK_UNIT } from '../../utils/grids
     styleUrls: ['./template-builder-background-columns.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush,
     standalone: true,
-    imports: [NgFor, NgStyle]
+    imports: [NgFor, NgStyle, NgClass]
 })
 export class TemplateBuilderBackgroundColumnsComponent {
+    @Input() show = true;
```

Comment:
Just show and hide the component in it parent. This component don't need this logic.